src/LoopbackModelBase.js

```diff
@@ -0,0 +1,62 @@
+const MicroserviceError = require('./MicroserviceError');
+
+module.exports = class LoopbackModelBase {
+    constructor({ modelName, model }) {
+        if (!modelName) {
+            throw LoopbackModelBase.createError(`No modelName was given to the
+                LoopbackModelBase constructor`);
+        }
+        if (!model) {
+            throw LoopbackModelBase.createError(`No Loopback Model Instance was given to the
+                LoopbackModelBase constructor`);
+        }
+
+        this.modelName = modelName;
+        this[this.modelName] = model;
```

Comment:
Don't see why you are assigning it to a variable property. It just makes to code less readable and might lead to collisions.